packages/utils/src/collections.ts

```diff
@@ -97,3 +97,11 @@ export function linkedListChanges(
   }
   return { lessers, greaters, list }
 }
+
+export function range(from: number, to: number, step: number) {
```

Comment:
lodash already has a range function, please use that

Reply:
Nice. Thanks! Updated.